### ISSUE TITLE
get() operation not accepting multiple subtree filters

### ISSUE THREAD

**Issue by neelimapp** [ncclient/ncclient#373]
I am trying to get RPC reply of multiple sub trees at once via get() operation as below but its giving me an error.
```python
getfacts='''
    <host-names xmlns="http://cisco.com/ns/yang/Cisco-IOS-XR-shellutil-cfg"/>
    <system-time xmlns="http://cisco.com/ns/yang/Cisco-IOS-XR-shellutil-oper"/>
    <interfaces xmlns="http://cisco.com/ns/yang/Cisco-IOS-XR-pfi-im-cmd-oper">
     <interfaces>
       <interface>
        <interface-name/>
       </interface>
      </interfaces>
     </interfaces>
    <inventory xmlns="http://cisco.com/ns/yang/Cisco-IOS-XR-invmgr-oper">
     <racks>
       <rack>
        <attributes>
            <inv-basic-bag>
                <software-revision/>
                <model-name/>
                <serial-number/>
            </inv-basic-bag>
        </attributes>
       </rack>
      </racks>
     </inventory>'''
  
def connect(host, port, user, password, source):
    conn = manager.connect(host=host,
                           port=port,
                           username=user,
                           password = password,
                           device_params={'name': 'iosxr'}
                           )

    rpc_reply = conn.get(filter=('subtree', getfacts)).xml
    print(rpc_reply)
```

Error:
```
return x if etree.iselement(x) else etree.fromstring(x.encode('UTF-8'), parser=_get_parser(huge_tree))
  File "src/lxml/etree.pyx", line 3235, in lxml.etree.fromstring
  File "src/lxml/parser.pxi", line 1876, in lxml.etree._parseMemoryDocument
  File "src/lxml/parser.pxi", line 1764, in lxml.etree._parseDoc
  File "src/lxml/parser.pxi", line 1127, in lxml.etree._BaseParser._parseDoc
  File "src/lxml/parser.pxi", line 601, in lxml.etree._ParserContext._handleParseResultDoc
  File "src/lxml/parser.pxi", line 711, in lxml.etree._handleParseResult
  File "src/lxml/parser.pxi", line 640, in lxml.etree._raiseParseError
  File "<string>", line 4
lxml.etree.XMLSyntaxError: Extra content at the end of the document, line 4, column 5
```

**Comment by einarnn** [comment]
@neelimapp, if you might have the time to look at this and proposing a modification, I can recommend you take a look at:

https://github.com/ncclient/ncclient/blob/b09e1a591f0a10befdbf430d133a9ecb598098d4/ncclient/operations/util.py#L48-L69

Something I thought might be a backwards compatible approach to supporting this is to allow the `spec` parameter to be a list of filters. This would only be valid for subtree filters, not xpath filters.

If you wish to take a look at this, I would happily review a proposed patch.

Cheers,

Einar

**Comment by einarnn** [comment]
Addressed by #397!
